Dockerfile.cuda

```diff
@@ -13,9 +17,11 @@ ENV GOARCH=$TARGETARCH
 RUN /usr/local/go/bin/go generate ./... \
     && /usr/local/go/bin/go build -ldflags "-linkmode=external -extldflags='-static' -X=github.com/jmorganca/ollama/version.Version=$VERSION -X=github.com/jmorganca/ollama/server.mode=release" .
 
-FROM nvidia/cuda:12.2.0-runtime-ubuntu22.04
+FROM $OS_NAME:$OS_VERSION
```

Comment:
keep full image names here instead making everything an `ARG`? much easier to read